processing/src/main/java/io/druid/query/lookup/LookupReferencesManager.java

```diff
@@ -103,44 +155,35 @@ public void start()
     if (!lifecycleLock.canStart()) {
       throw new ISE("can't start.");
     }
-
     try {
       LOG.info("LookupReferencesManager is starting.");
-
-      loadSnapshotAndInitStateRef();
-
+      loadAllLookupsAndInitStateRef();
       if (!testMode) {
         mainThread = Execs.makeThread(
             "LookupReferencesManager-MainThread",
-            new Runnable()
-            {
-              @Override
-              public void run()
-              {
-                try {
-
-                  if (!lifecycleLock.awaitStarted()) {
-                    LOG.error("WTF! lifecycle not started, lookup update notices will not be handled.");
-                    return;
-                  }
+            () -> {
+              try {
+                if (!lifecycleLock.awaitStarted()) {
+                  LOG.error("WTF! lifecycle not started, lookup update notices will not be handled.");
+                  return;
+                }
 
-                  while (!Thread.interrupted() && lifecycleLock.awaitStarted(1, TimeUnit.MILLISECONDS)) {
-                    try {
-                      handlePendingNotices();
-                      LockSupport.parkNanos(LookupReferencesManager.this, TimeUnit.MINUTES.toNanos(1));
-                    }
-                    catch (Throwable t) {
-                      LOG.makeAlert(t, "Error occured while lookup notice handling.").emit();
-                    }
+                while (!Thread.interrupted() && lifecycleLock.awaitStarted(1, TimeUnit.MILLISECONDS)) {
+                  try {
+                    handlePendingNotices();
+                    LockSupport.parkNanos(LookupReferencesManager.this, TimeUnit.MINUTES.toNanos(1));
+                  }
+                  catch (Throwable t) {
+                    LOG.makeAlert(t, "Error occured while lookup notice handling.").emit();
```

Comment:
is this just indentation change? ... why did it change ?

Reply:
The indentation changed when I changed the Runnable right above it.